ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -373,6 +374,13 @@ def serialize_value(self, display_context: "WorkflowDisplayContext", value: Base
                 "input_variable_id": str(workflow_input_display.id),
             }
 
+        if isinstance(value, StateValueReference):
+            state_value_display = display_context.global_state_value_displays[value]
+            return {
+                "type": "STATE_VALUE",
+                "state_value_id": str(state_value_display.id),
+            }
+
```

Comment:
[blocking] We should match the schema represented by: https://github.com/vellum-ai/vellum/blob/main/django/app/domain_models/types/workflows.py#L676-L679